test/files/test-e2e.sh

```diff
@@ -0,0 +1,22 @@
+#! /bin/bash
+
```

Comment:
It would be helpful to run this script with `set -euxo pipefail` so we can see where it is complaining

Somewhere near the top you need to do something like this: https://github.com/openshift/origin-aggregated-logging/blob/master/hack/testing/templates/logging-ci-test-runner-template.yaml#L47
```bash
cp /tmp/test/config /tmp/kubeconfig
export KUBECONFIG=/tmp/kubeconfig
```
also, it's likely that the namespace is coming from the `KUBECONFIG` - it's probably the current context in the kubeconfig file.

I'm not sure why the logging CI test doesn't keep the reference to the old namespace.  I'm assuming it is because it does `oc project openshift-logging` fairly early in the test.  You could do something similar by doing a 
```
oc project $TEST_NAMESPACE
```
right after exporting the `KUBECONFIG` env var

Reply:
not sure it is the `KUBECONFIG` thing - how did it reference the ci namespace otherwise?  maybe it is that the kubeconfig file has to be writable?  since /tmp/test/config is mounted directly from the secret, it isn't writable - that's why logging had to make a copy
Also, I'm not sure how the go operator sdk test is pulling the elasticsearch-operator image from the api.ci - it may need a pull secret, and I'm not sure where that comes from